README.md

```diff
@@ -44,30 +50,30 @@ role **roles/iam.serviceAccountTokenCreator** on `GSA_SIGNER`.
 #### Example
 
 ```elixir
-    iex> access_token = Goth.Token.for_scope("https://www.googleapis.com")
-    iex> oauth_config = %GcsSignedUrl.SignBlob.OAuthConfig{service_account: "project@gcs_signed_url.iam.gserviceaccount.com", access_token: access_token}
-    iex> GcsSignedUrl.generate_v4(oauth_config, "my-bucket", "my-object.jpg", verb: "PUT", expires: 1800, headers: ["Content-Type": "application/jpeg"])
-    {:ok, "https://storage.googleapis.com/my-bucket/my-object.jpg?X-Goog-Expires=1800..."}
+iex> access_token = Goth.Token.for_scope("https://www.googleapis.com")
+iex> oauth_config = %GcsSignedUrl.SignBlob.OAuthConfig{service_account: "project@gcs_signed_url.iam.gserviceaccount.com", access_token: access_token}
+iex> GcsSignedUrl.generate_v4(oauth_config, "my-bucket", "my-object.jpg", verb: "PUT", expires: 1800, headers: ["Content-Type": "application/jpeg"])
+{:ok, "https://storage.googleapis.com/my-bucket/my-object.jpg?X-Goog-Expires=1800..."}
 ```
 
 ### On-Premise Signing
 
 In this scenario you have a service account key in form of a JSON file on your machine. The library will use the
 private key to create the signature, no network calls are needed.
 
-1. Load the client
-```elixir
-iex> GcsSignedUrl.Client.load_from_file("/home/alexandrubagu/config/google.json")
-```
-or 
-
-```elixir
-iex> service_account = service_account_json_string |> Jason.decode!
-iex> GcsSignedUrl.Client.load(service_account)
-```
- 
- 2. Generate signed url (V2)
- ```elixir
- GcsSignedUrl.generate(client, "my-bucket", "my-object.mp4")
- GcsSignedUrl.generate(client, "my-bucket", "my-object.mp4", expires: GcsSignedUrl.hours_after(3))
- ```
+1.  Load the client
+    ```elixir
+    iex> GcsSignedUrl.Client.load_from_file("/home/alexandrubagu/config/google.json")
+    ```
+or
+
+    ```elixir
```

Comment:
This does not render well... see the rich diff!

Reply:
> This does not render well... see the rich diff!

See if this still an issue?

Through VsCode.
![image](https://user-images.githubusercontent.com/134518/96602092-38a53180-1325-11eb-9f9a-7825c5def772.png)

Through `ex_doc`.
![image](https://user-images.githubusercontent.com/134518/96602191-55da0000-1325-11eb-846c-8029e9742605.png)